fix hidden (overlayed) tooltip

berlic opened the pull request:
moved component's icons tooltip to the right

before:
![2016-04-17 13 13 54](https://cloud.githubusercontent.com/assets/2950241/14586503/e9b87d74-04a2-11e6-8e7c-e26c1b0525a6.png)

after:
![2016-04-17 13 14 14](https://cloud.githubusercontent.com/assets/2950241/14586508/f4d519b0-04a2-11e6-90b7-ffcdcf188a03.png)

jbrooksuk commented:
This doesn't actually properly fix the tooltip, it just moves it so that it's less hidden.

The CSS itself is what needs fixing here I'm afraid.

berlic commented:
I've replaced the commit with a proper fix, I think.
Changed [tooltip container](http://getbootstrap.com/javascript/#tooltips-options) to 'body'.

Looks like this:
![2016-04-17 22 59 34](https://cloud.githubusercontent.com/assets/2950241/14589926/a9a2b5ca-04f6-11e6-8570-f8b69130feb9.png)

jbrooksuk commented:
This is perfect! Could you rebase this please.

berlic commented:
Rebased

jbrooksuk commented:
Thank you!
